api/v1alpha1/podtypes/access_policy.go

```diff
@@ -3,13 +3,14 @@ package podtypes
 // +kubebuilder:object:generate=true
 type AccessPolicy struct {
 	//+kubebuilder:validation:Optional
-	Inbound InboundPolicy `json:"inbound,omitempty"`
+	Inbound *InboundPolicy `json:"inbound,omitempty"`
 	//+kubebuilder:validation:Optional
 	Outbound OutboundPolicy `json:"outbound,omitempty"`
```

Comment:
Is there a similar issue with OutboundPolicy?

Reply:
No rules on OutboundPolicy are required, so not as far as I can tell. @esphen